tests/format/scss/trailing-comma/__snapshots__/format.test.js.snap

```diff
@@ -190,6 +294,50 @@ a {
 ================================================================================
 `;
 
+exports[`issue-6920.scss - {"trailingComma":"es5"} format 1`] = `
+====================================options=====================================
+parsers: ["scss"]
+printWidth: 80
+trailingComma: "es5"
+                                                                                | printWidth
+=====================================input======================================
+$my-map: (
+  'foo': 1, // Comment
+  'bar': 2, // Comment
+);
+
+=====================================output=====================================
+$my-map: (
+  "foo": 1,
+  // Comment
+  "bar": 2, // Comment
+);
```

Comment:
This is actually wrong, the first comment should not print on separate line, but it's a different issue, we can deal with it later.

Reply:
OK, I thought it is the default behavior since it was like this in the released version.

Reply:
It's bug, but we don't have a comment attach logic in CSS printer, it's not easy to fix(I guess).

Reply:
I could be wrong, if we always print comments with other elements instead of a separate element, it should be fixed? Don't know.